core/src/test/java/org/infinispan/persistence/ClusterCacheLoaderTest.java

```diff
@@ -23,7 +23,7 @@
  * @author Mircea.Markus@jboss.com
  */
 @Test(groups = "functional", testName = "persistence.ClusterCacheLoaderTest")
-@InCacheMode({ CacheMode.INVALIDATION_SYNC, CacheMode.DIST_SYNC, CacheMode.REPL_SYNC, CacheMode.SCATTERED_SYNC })
+@InCacheMode({ CacheMode.INVALIDATION_SYNC/*, CacheMode.DIST_SYNC, CacheMode.REPL_SYNC, CacheMode.SCATTERED_SYNC*/ })
```

Comment:
Unstable? Please add an explanation

Reply:
Didn't mean to comment these out, thanks.